pkgs/build-support/dotnet/build-dotnet-module/default.nix

```
@@ -118,23 +118,23 @@ in stdenvNoCC.mkDerivation (args // {
 
     fetch-deps = writeScript "fetch-${pname}-deps" ''
       set -euo pipefail
```

This was also an issue before this PR, but I think all the store paths end up making the generated script a bit difficult to read. I think we should add them all to `PATH` instead with something like this:
```suggestion
      set -euo pipefail

      export PATH="${lib.makeBinPath [ coreutils dotnet-sdk etc ]}"
```

That way we just generate one big line containing all the store paths, and the rest of the script should be a lot more readable. By unsetting `PATH` we also ensure we never accidentally use any dependencies not provided by the script itself.